package.json

```diff
@@ -7,25 +7,25 @@
     "cover": "grunt coverage"
   },
   "dependencies": {
-    "async": "0.1.x",
+    "async": "^1.5.2",
     "debug": "^2.1.1",
-    "nconf": "~0.6.7"
+    "nconf": "^0.8.4"
   },
   "main": "index.js",
   "registerAs": "drivex",
   "peerDependencies": {
-    "nemo": "^0.4.0",
+    "nemo": "^2.1.1",
```

Comment:
i assume you need to update the nemo-locatex dependency before the CI will succeed